src/components/projectRegister/StepThreeForm.tsx

```
@@ -587,46 +625,52 @@ export default function StepThreeForm({ initialData }: StepThreeFormProps) {
+              <div
+                key={account.id}
+                onClick={() => handleSelectAccount(account.id)}
+                className={`rounded-xl border p-4 cursor-pointer transition-colors ${
+                  selectedAccountId === account.id
+                    ? "border-main-color bg-secondary-color"
+                    : "border-gray-300 hover:bg-gray-50"
+                }`}
+              >
+                <div className="flex items-center justify-between">
+                  <div className="flex items-center gap-3">
+                    {/* 라디오 버튼 추가 */}
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)